libraries/ms-common/src/x86/adc.c

```diff
@@ -176,32 +178,200 @@ StatusCode adc_read_converted(AdcChannel adc_channel, uint16_t *reading) {
   return STATUS_CODE_OK;
 }
 
-// the following functions are wrappers over the legacy AdcChannel API dealing with GpioAddresses
```

Comment:
seems like your branch might be outdated (these shouldn't be being deleted), maybe merge `mpxe` in or rebase